src/components/home/Socials.tsx

```diff
@@ -23,11 +24,41 @@ const Socials = () => {
         className="w-full md:hidden"
       />
 
-      <FaDiscord className="absolute left-[13.5%] top-[52%] duration-200 hover:scale-[110%] hover:cursor-pointer xl:left-[13.5%] xl:top-[45%] xl:text-8xl 2xl:left-[14%] 2xl:top-[46%] 2xl:text-9xl" />
-      <FaTiktok className="absolute bottom-[7%] left-[32.5%] duration-200 hover:scale-[110%] hover:cursor-pointer xl:bottom-[7%] xl:left-[31.5%] xl:text-8xl 2xl:bottom-[7%] 2xl:left-[32%] 2xl:text-9xl" />
-      <FaInstagram className="absolute left-[51.5%] top-[53%] duration-200 hover:scale-[110%] hover:cursor-pointer xl:left-[48.5%] xl:top-[45%] xl:text-8xl 2xl:left-[49%] 2xl:top-[46%] 2xl:text-9xl" />
-      <FaYoutube className="absolute bottom-[6.5%] right-[21%] duration-200 hover:scale-[110%] hover:cursor-pointer xl:bottom-[7%] xl:right-[27%] xl:text-8xl 2xl:bottom-[7%] 2xl:right-[27.5%] 2xl:text-9xl" />
-      <FaFacebookF className="absolute right-[4%] top-[53%] duration-200 hover:scale-[110%] hover:cursor-pointer xl:right-[12.5%] xl:top-[45%] xl:text-8xl 2xl:right-[13%] 2xl:top-[46%] 2xl:text-9xl" />
+      <a
```

Comment:
Don't use `a` tags. Use `Link` component from `next/link`
Check footer for an example